frontend/src/lib/components/AccessControlAction.tsx

```diff
@@ -36,18 +38,27 @@ export const accessLevelSatisfied = (
 
 export const getAccessControlDisabledReason = (
     resourceType: AccessControlResourceType,
-    userAccessLevel: AccessControlLevel | undefined,
     minAccessLevel: AccessControlLevel,
+    userAccessLevel?: AccessControlLevel,
     includeAccessDetails: boolean = true
 ): string | null => {
-    const hasAccess = userAccessLevel ? accessLevelSatisfied(resourceType, userAccessLevel, minAccessLevel) : true
+    // If the userAccessLevel is not provided, we use the app context for that resource type
+    const parsedUserAccessLevel = userAccessLevel ?? getAppContext()?.resource_access_control?.[resourceType]
```

Comment:
Note: in the future this may also need to support `effective_resource_access_control` so we may need to add a prop so the parent can decide which one is used but should be ok for now.

Reply:
I don't know what the `effective_*` access control is but yeah, it's reasonable that we'll have to change this in the future